### PR TITLE
feat: Avatar and documentation

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
Added avatar component and documentation related to colors, installation and Avatar

### CODE REVIEW

File: CSS/nav.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+       height: 10vh;
+   }
+   
+   .header-icon {
+       font-size: xx-large;
+       font-family: 'Kalam', cursive;
+   }
+   
+   .nav-links {
+       width: 300px;
```
Comment (Jia-A) [inline CSS/nav.css:12]: Maybe change px to rems.

File: Docs/colors.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Colors | LightUI</title>
+
+    <link rel="stylesheet" href="doc.css">
+</head>
+
+<body>
+    <div class="nav flex-space-between padding-xxl shadow-sm">
+        <!-- <ion-icon class="header-icon" name="sunny-outline"></ion-icon> -->
```
Comment (Jia-A) [inline Docs/colors.html:15]: you can remove the commented code.

**Review by FarhanMobashir — COMMENTED** [review]
✔️✔️

File: Docs/doc.css [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    
+    .danger {
+        background-color: #F44336;
+    }
+    
+    .success {
+        background-color: #4CAF50;
+    }
+    
+    .warning {
```
Comment (Jia-A) [inline Docs/doc.css:110]: You can make variables for these colors as they are gonna be used twice or thrice in whole library.

File: Docs/doc.css [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        /* display: none; */
+        width: 5px;
+    }
+    
+    body::-webkit-scrollbar-thumb {
+        background-color: var(--prime-blue);
+    }
+    /* body {
+        scrollbar-width: none;
+        -ms-overflow-style: none;
```
Comment (Jia-A) [inline Docs/doc.css]: remove these comments for clean code.

File: Docs/docs.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Documentation | Light UI</title>
+    <link rel="stylesheet" href="doc.css">
+
+</head>
+
+<body>
+    <div class="nav flex-space-between padding-xxl shadow-sm">
+        <!-- <ion-icon class="header-icon" name="sunny-outline"></ion-icon> -->
```
Comment (Jia-A) [inline Docs/docs.html:15]: remove this commented code.

**Review by Jia-A — COMMENTED** [review]
Everything looks good, just change the px to rems in some areas, and remove the commented code.

File: CSS/nav.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply CSS/nav.css:12]: Thankyou, will do the changes

File: Docs/colors.html (hunk 2 above)
Comment (MaruthiPothuganti) [reply Docs/colors.html:15]: This icon is related to switching between dark and white mode. As I haven't added the functionality, I just commented it out.

File: Docs/doc.css (hunk 4 above)
Comment (MaruthiPothuganti) [reply Docs/doc.css]: Thankyou, will make the change.

**Pull request merged**